t/errlog.t

```diff
@@ -932,3 +932,65 @@ maybe log lines: #1
 end
 \z
 --- skip_nginx: 2: <1.11.2
+
+
+
+=== TEST 23: the system default filter level is "debug"
```

Comment:
@membphis 
I see. Now I change `TEST 25` to test `get_sys_filter_level` under `init_by_lua*`.
Is it ok?